TestFoundation/TestNSArray.swift

```diff
@@ -302,4 +304,48 @@ class TestNSArray : XCTestCase {
         let emptyArray = NSArray().sortedArrayWithOptions([]) { _,_ in .OrderedSame }
         XCTAssertTrue(emptyArray.isEmpty)
     }
+
+    func test_sortUsingFunction() {
+        let inputNumbers = [11, 120, 215, 11, 1, -22, 35, -89, 65]
+        let mutableInput = inputNumbers.bridge().mutableCopy() as! NSMutableArray
```

Comment:
This currently fails due to a secondary failure of NSArray's mutable copy not returning the correct value on linux